docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/enable-plugin.md

```diff
@@ -0,0 +1,48 @@
+---
+title: v4 plugin migration - Enable a plugin - Strapi Developer Docs
+description:
+canonicalUrl:
+prev: /developer-docs/latest/update-migration-guides/migration-guides/v4/plugin-migration.md
+---
+
+<!-- TODO: update SEO -->
+
+# v4 plugin migration: Enable a plugin
+
+!!!include(developer-docs/latest/update-migration-guides/migration-guides/v4/snippets/plugin-migration-intro.md)!!!
+
+<br/>
+
+::: callout
+A Strapi v3 plugin was enabled if it was manually installed or found in the `plugins` directory.
+
+In Strapi v4:
+
+- If a plugin is installed via the [automatic plugins discovery](/developer-docs/latest/plugins/plugins-intro.md#automatic-plugins-discovery) feature, it is automatically enabled.
```

Comment:
I don't think we can say that the discovery feature doesn't install plugins. A proposition:
```
If a plugin is installed, then it will automatically be discovered via the [automatic plugins discovery](/developer-docs/latest/plugins/plugins-intro.md#automatic-plugins-discovery) feature and will be considered as enabled by default.
```